### PR TITLE
Tel link and improve UniversalLink

### PR DESCRIPTION
**Pull request by giuliaghisini**
added the ability to enter phone numbers in links in draftJs and UrlWidget.

Fixed UniversalLink component to work with emails and tel link

### CODE REVIEW

**Review by nzambello — APPROVED** [review]
LGTM, nice job

File: src/components/manage/AnchorPlugin/utils/URLUtils.js [hunk 1]
```diff
@@ -11,13 +12,24 @@ export default {
...
+  },
+
   normaliseMail(email) {
     if (email.toLowerCase().startsWith('mailto:')) {
       return email;
     }
     return `mailto:${email}`;
   },
 
+  normalizeTelephone(tel) {
```
Comment (tiberiuichim) [inline src/components/manage/AnchorPlugin/utils/URLUtils.js:26]: normalise or normalize? I see both versions in this code.

File: src/helpers/Url/Url.js [hunk 2]
```diff
@@ -159,7 +159,9 @@ export function flattenHTMLToAppURL(html) {
  * @returns {string} New URL with app
  */
 export function addAppURL(url) {
-  return `${settings.apiPath}${url}`;
+  return url.indexOf(settings.apiPath) === 0
```
Comment (tiberiuichim) [inline src/helpers/Url/Url.js:162]: Is it relevant here to also use settings.internalApiPath? We use both of them, as frontend comunicates with the backend through settings.internalApiPath.

**Review by tiberiuichim — CHANGES_REQUESTED** [review]

**Review by tiberiuichim — APPROVED** [review]

**Comment by cekk** [comment]
@tisto @sneridagh is it ok for you?

**Review by tiberiuichim — COMMENTED** [review]
The LinkView starts to get complicated with lots of ifs and by implementing "some" use cases we open the question: what if I have other cases?

Can we make this configurable with a registry of "link parsers to component renderer"?

**Comment by giuliaghisini** [comment]
@sneridagh @tisto can we merge in master?

**Review by sneridagh — APPROVED** [review]
LGTM, @cekk @giuliaghisini didn't this was related to some change in restapi needed to be merged? If it's already merged we can do it right away.

**Comment by cekk** [comment]
I don't find anything related in my restapi prs, so it's a green button from me.

**Pull request merged**